schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+                 "type": "number",
+		 "minItems": 1,
+                 "maxItems": 3
+            }
+       },
+       "OPTICAL_DISTORTION": {
+           "description": "Information needed to convert from undistorted to distorted image plane",
+           "type": "object",
+	   "properties" : {
+	      "coefficients": {
+	         "type": "array",
+		 "items": {
+		    "type": "number"
+		  }
+		},
+	        "key" : {
+		    "type": "string",
+		    "description": "key into LUT to determine equation to use"
+		    }
+	 },
+	"required": ["coefficients", "equation_key"]
+       },
+
+       "STARTING_DETECTOR_SAMPLE": {
+           "description": "The sample on the detector where the data starts",
+           "type": "number"
+       },
+       "STARTING_DETECTOR_LINE": {
+           "description": "The line on the detector where the data starts",
+           "type": "number"
+       },
+       "FOCAL2PIXEL_X": {
+           "description": "Transformation x coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "FOCAL2PIXEL_Y": {
+           "description": "Transformation y coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "SENSOR_LOCATION": {
+           "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+      },
+       "SENSOR_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+	     },
+       "SUN_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
```

Comment:
We should note the quaternion format. The "scalar" quaternion is the first, then the three "rotation" quaternions. This is different from what NAIF returns with SPICE. The line scan model uses the Alternate Style described in this NAIF white paper ftp://naif.jpl.nasa.gov/pub/naif/misc/Quaternion_White_Paper/Quaternions_White_Paper.pdf.

Reply:
:+1: